src/views/ManageList.jsx

```diff
@@ -1,7 +1,61 @@
-export function ManageList() {
+import { addItem } from '../api/firebase';
+import { useState } from 'react';
+
+export function ManageList({ listPath }) {
+	const [message, setMessage] = useState('');
+	const [userItem, setUserItem] = useState('');
+	// Presetting item duration to 7, as that option starts
+	// selected on page rendering.
+	const [itemDuration, setItemDuration] = useState(7);
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		try {
+			await addItem(listPath, {
+				itemName: userItem,
+				daysUntilNextPurchase: itemDuration,
+			});
+			setMessage('Item added to the list!');
+		} catch (error) {
+			setMessage('Unable to add item to the list.');
+			console.log(error);
```

Comment:
would set this to `console.error(error)` since it provides more information on the error.

Reply:
Good idea! I've been meaning to dig more into more console methods; thank you for reminding me!

Changed in a recent push!